lib/cli.js

```diff
@@ -119,7 +119,15 @@ exports.run = () => {
 	}
 
 	if (cli.flags.concurrency === '') {
-		throw new Error(colors.error(figures.cross) + ' The --concurrency and -c flags must be provided the maximum number of test files to run at once.');
+		throw new Error(colors.error(figures.cross) + ' The --concurrency and -c flags must be provided.');
+	}
+
+	if (cli.flags.concurrency && isNaN(parseInt(cli.flags.concurrency, 10))) {
```

Comment:
I think this should be `Number.isInteger(Number.parseFloat('cli.flags.concurrency'))`. That way even values like `9.5` are rejected, which parse to `9` with `parseInt()`.